### ISSUE TITLE
training error issue?

### ISSUE THREAD

**Issue by fzr2009** [ashishrana160796/TextSummarizer#1]
Hey Ashish,
I hope you're doing fine, please elaborate what issues are you getting in training file.

**Comment by ashishrana160796** [comment]
Hi,
Currently the error state is `FailedPreconditionError` which we have reached from `ResourceExhausted Error` earlier on. Resource Exhausted can clearly be attributed to large batch size while training on smaller machine, after reducing this it is removed. 

`FailedPreconditionError` now related to `In [27]` --> `embedding_1` in train.ipynb  we are currently working on its removal. Error messages in the train.ipynb explains it. 

I hope it clears your doubt & apologies for late reply. Lets keep this open until error message is resolved.

-Thanks!

**Comment by fzr2009** [comment]
try this:
model = Sequential()
model.add(Embedding(vocab_size, embedding_size,
                    input_length=maxlen,
                    W_regularizer=regularizer, dropout=p_emb, weights=[embedding], mask_zero=True,
                    name='embedding_1'))
for i in range(rnn_layers):
    lstm = LSTM(rnn_size, return_sequences=True, # batch_norm=batch_norm,
                W_regularizer=regularizer, U_regularizer=regularizer,
                b_regularizer=regularizer, dropout_W=p_W, dropout_U=p_U,
                name='lstm_%d'%(i+1)
                  )
    model.add(lstm)
    model.add(Dropout(p_dense,name='dropout_%d'%(i+1)))

**Comment by fzr2009** [comment]
model = Sequential()
model.add(Embedding(vocab_size, embedding_size,
                    input_length=maxlen,
                    W_regularizer=regularizer, dropout=p_emb, weights=[embedding], mask_zero=True,
                    name='embedding_1'))
for i in range(rnn_layers):
    lstm = LSTM(rnn_size, return_sequences=True, # batch_norm=batch_norm,
                W_regularizer=regularizer, U_regularizer=regularizer,
                b_regularizer=regularizer, dropout_W=p_W, dropout_U=p_U,
                name='lstm_%d'%(i+1)
                  )
    model.add(lstm)
    model.add(Dropout(p_dense,name='dropout_%d'%(i+1)))

**Comment by fzr2009** [comment]
or change backend to theano it resolved some issues

**Comment by ashishrana160796** [comment]
Thanks for solution @fzr2009. Also, we are interested in using tensor-flow backend for keeping consistency throughout the software of which this `rep` is a submodule.

-Thanks!
